ui/tests/integration/components/secret-engine/configure-wif-test.js

```diff
@@ -0,0 +1,901 @@
+          this.config = this.store.createRecord('azure/config');
+          this.config.backend = this.id;
+
+          await render(hbs`
+                <SecretEngine::ConfigureWif @backendPath={{this.id}} @displayName={{this.displayName}} @type='azure' @model={{this.config}} @issuerConfig={{this.issuerConfig}}/>
+              `);
+          await fillInAzureConfig('azure');
+          await click(SES.wif.accessType('wif'));
+          await fillInAzureConfig('withWif');
+          await click(SES.wif.accessType('azure'));
+
+          assert
+            .dom(GENERAL.toggleInput('Root password TTL'))
+            .isNotChecked('rootPasswordTtl is cleared after toggling accessType');
+          assert
+            .dom(GENERAL.inputByAttr('clientSecret'))
+            .hasValue('', 'clientSecret is cleared after toggling accessType');
+
+          await click(SES.wif.accessType('wif'));
+          assert
+            .dom(GENERAL.inputByAttr('issuer'))
+            .hasValue('', 'issuer shows no value after toggling accessType');
+          assert
+            .dom(GENERAL.inputByAttr('issuer'))
+            .hasAttribute(
+              'placeholder',
+              'https://vault-test.com',
+              'issuer shows no value after toggling accessType'
+            );
+          assert
+            .dom(GENERAL.inputByAttr('identityTokenAudience'))
+            .hasValue('', 'idTokenAudience is cleared after toggling accessType');
+          assert
+            .dom(GENERAL.toggleInput('Identity token TTL'))
+            .isNotChecked('identityTokenTtl is cleared after toggling accessType');
+        });
+
+        test('it shows the correct access type subtext', async function (assert) {
+          this.id = `azure-${this.uid}`;
+          this.displayName = 'Azure';
+          this.issuerConfig = createConfig(this.store, this.id, 'issuer');
+          this.config = this.store.createRecord('azure/config');
+          this.config.backend = this.id;
+
+          await render(hbs`
+                <SecretEngine::ConfigureWif @backendPath={{this.id}} @displayName={{this.displayName}} @type='azure' @model={{this.config}} @issuerConfig={{this.issuerConfig}}/>
+              `);
+
+          assert
+            .dom(SES.wif.accessTypeSubtext)
+            .hasText(
+              'Choose the way to configure access to Azure. Access can be configured either using an Azure account credentials, or with the Plugin Workload Identity Federation (WIF).'
+            );
+        });
+      });
+
+      module('AWS specific', function () {
+        test('it clears access type inputs after toggling accessType', async function (assert) {
+          this.id = `aws-${this.uid}`;
+          this.displayName = 'AWS';
+          this.issuerConfig = createConfig(this.store, this.id, 'issuer');
+          this.config = this.store.createRecord('aws/root-config');
+          this.config.backend = this.id;
```

Comment:
Can we consolidate this and instead of doing this in every test, do so in the beforeEach hook for this module?